src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Approve.tsx

```diff
@@ -200,7 +200,7 @@ export const Approve: React.FC<FoxFarmingApproveProps> = ({ accountId, onNext })
       loading={state.loading}
       loadingText={translate('common.approve')}
       preFooter={preFooter}
-      providerIcon='https://assets.coincap.io/assets/icons/256/fox.png'
+      providerIcon='https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025'
```

Comment:
```suggestion
      providerIcon='https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025'
```

Reply:
Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)